package.json

```diff
@@ -13,8 +13,6 @@
     "watch:test:unit": "nodemon --exec \"npm run test:unit\" ./test ./app ./ui",
     "test:unit": "cross-env METAMASK_ENV=test mocha --exit --require test/setup.js --recursive \"test/unit/**/*.js\" \"ui/app/**/*.test.js\" && dot-only-hunter",
     "test:single": "cross-env METAMASK_ENV=test mocha --require test/helper.js",
-    "test:integration": "npm run test:integration:build && npm run test:flat && npm run test:mascara",
```

Comment:
The issue with this is that the `test-integration-flat-firefox/chrome` is also used for `test-integration-mascara`